data/json/items/tool/musical_instruments.json

```
@@ -61,6 +61,28 @@
       ]
     }
   },
+  {
+    "id": "wood_flute",
+    "type": "TOOL",
+    "name": { "str": "wooden flute" },
+    "copy-from": "bone_flute",
```

Also needs a description change.
```suggestion
    "copy-from": "bone_flute",
    "description": "A simple flute carved from wood with five finger holes.",
```